### ISSUE TITLE
a recipes for Prisma

### ISSUE THREAD

**Issue by soqt** [avajs/ava#2014]
Would someone please create a recipes for Prisma test environment? Prisma is an ORM for graphql development. However when try to clean up database  using Prisma client, it throws an error "only HTTP(S) protocol is supported". It would be nice to have a recipes for projects using Prisma.

**Comment by novemberborn** [comment]
Could you share a bit about how you're using AVA with Prisma?

**Comment by soqt** [comment]
I just realized it has nothing to do with Prisma. I have `prisma server` and `API server` both in separate docker containers. I tried to run `docker-compose exec APIServer ava`, which runs test from `API server container`. However, if there is a request that requesting `http://prisma`, an error throws "Only HTTP(S) protocol is supported". I tried to run `ava` locally and everything is good.   Is there anyway to run Ava inside a docker container, (and ava needs to request other containers)?
